packages/extension-ui/src/Popup/CreateAccount/CreateAccount.spec.tsx

```diff
@@ -47,6 +47,17 @@ describe('Create Account', () => {
     );
 
   describe('CreateAccount component', () => {
+    const clickNext = () => wrapper.findWhere(
+      (node) => node.type() === 'button' && node.text() === 'Next'
+    ).simulate('click');
+
+    const clickBack = () => wrapper.findWhere(
+      (node) => node.type() === 'button' && !!node.find('.arrowLeft').length
+    ).simulate('click');
+
+    const isRendered = <T,>(statelessComponent: FunctionComponent<T>) => expect(wrapper.find(statelessComponent).exists()).toBeTruthy();
+    const isUnmounted = <T,>(statelessComponent: FunctionComponent<T>) => expect(wrapper.find(statelessComponent).exists()).toBeFalsy();
```

Comment:
How about
```suggestion
    const isPresent = <T,>(statelessComponent: FunctionComponent<T>) => expect(wrapper.find(statelessComponent).exists()).toBeTruthy();
    const isAbsent = <T,>(statelessComponent: FunctionComponent<T>) => expect(wrapper.find(statelessComponent).exists()).toBeFalsy();
```
to (or sth like that) to be consistent between the two twin functions?